software/gamelogic/gamelogic.cpp

```diff
@@ -57,9 +57,11 @@ int main () {
             }
         }
         else { // dealing with UI / BT
+            cout << "i'm here" << endl;
             Init_RS232();

```

Comment:
I want to confirm, these functions are "safe" if there is only one RFS chip?

Reply:
I don't have an answer to this. Have to ask @Eleiah or @mslongo

Reply:
I think I have tested Init_RS232() with only one card. It still worked. I could just move the card manually and the other port would work.
If this isn't what you're talking about then idk @paullin03